### ISSUE TITLE
Scrap LinkedIn company page

### ISSUE THREAD

**Issue by emonnaim** [dibpedro/linkedin_webscraping#1]
can I scrap the LinkedIn company page about section? Could you kindly assist?

**Comment by dibpedro** [comment]
Yes, you can. Although, I think you may have to be logged in with some account to do so. When you click on some job link, you are redirected to a page where the "About" section of the company is shown.
With Selenium you can click on the "Show more" button (it's not mandatory to do it, but then you'll have to find another class where the text is in) and then you can get the text (with BeautifulSoup or Selenium) from the "inline-show-more-text" class, which is inside the "jobs-company__box" class. I suggest you to inspect the page and see what you can get from it, as some kind of training. 
Hope I was helpful.
